CHANGELOG.md

```
@@ -1,50 +1,33 @@
 # [2.0.0](https://github.com/VilnaCRM-Org/user-service/compare/v1.8.0...v2.0.0) (2024-11-25)
 
-
 ### Documentation
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)